dear_petition/petition/export/documents/records_summary.py

```diff
@@ -55,11 +55,23 @@ def generate_context(batch, attorney, client):
     # sort tables first by county, then by jurisdiction
     tables_keys.sort()
     tables = []
-    for i, k in enumerate(tables_keys, start=1):
-        offense_records = tables_data[k]
+    for table_index, table_key in enumerate(tables_keys, start=1):
+        # get offense records for county and jurisdiction
+        offense_records = tables_data[table_key]
+
         # sort offense records by file number
         offense_records.sort(key=lambda x: x["file_no"])
-        table = {"county": k[0], "jurisdiction": k[1], "idx": i, "offense_records": offense_records}
+
+        # assign index for each offense record
+        for offense_record_index, offense_record in enumerate(offense_records, start=1):
+            offense_record["idx"] = offense_record_index
```

Comment:
Is the template doing `idx + 1` as the count? This is a bit of a personal opinion, but I feel like we should keep as much python logic out of the template file as possible because it's a binary file that we can't properly track in git

Reply:
I agree about keeping the logic out of the template.  The template uses the idx value as is. `enumerate(offense_records, start=1)` ensures the index starts at 1 and not 0.

Reply:
Realized I forgot to add checks in the unit tests.  I'll do that.